server/engine/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -30,8 +30,8 @@ db2SqlParser.maxIntValue=%s not allowed. Allowed range is 1 to 32767
 db2SqlParser.pieceSize=%s not allowed. Size must be in KB, MB or GB
 db2SqlParser.size=%s not allowed. Size must be in GB
 parsers.validValueMsg=%s not allowed. It must be %s
-parsers.maxLength=Max length limit of %s bytes allowed for %s.
-parsers.exactLength=Exact length of %s must be %s bytes
+parsers.maxLength=Max length limit of %s characters allowed for %s.
```

Comment:
Maximum length of %s is 30 characters.
Maximum length of 30 characters for %s.
No need to repeat max, limit and allowed.

Reply:
updated as per suggestion